.travis.yml

```diff
@@ -45,6 +45,15 @@ matrix:
     env: WP_TRAVISCI="yarn lint"
   - language: node_js
     env: WP_TRAVISCI="yarn test-dangerci-and-adminpage"
+  # We can't test PHP 7.3 with WP_BRANCH=previous just yet, as previous is currently 4.9.9.
```

Comment:
Since WordPress 5.1 will be out soon, I wonder if it isn't best to wait and then remove all this in a couple of weeks.

Reply:
If it's better to wait until Feb 21st for WP 5.1 before merging this?

I'm pretty indifferent about it... Suppose it isn't much work to change this in a separate PR in a week. :-)

How ever you prefer! :-)

Reply:
You're right. I'll merge this now. The more tests the better, even if it's only for a couple of weeks.

Reply:
done in #11398